docs/guides/transpile-rest-api.mdx

```diff
@@ -15,17 +15,17 @@ You have two transpilation options:
 
 * Transpile [locally via Qiskit](./defaults-and-configuration-options) before generating the QASM string.
 
-* Use the [Qiskit Transpiler Service API](https://cloud-transpiler.quantum.ibm.com/docs).
+* Use the [Qiskit Transpiler Service Python client](/api/qiskit-ibm-transpiler) or [Qiskit Transpiler Service REST API](/api/qiskit-ibm-transpiler-rest).
 
 <Admonition type="note">
-    [Transpilation is necessary](https://docs.quantum.ibm.com/announcements/product-updates/2024-02-14-qiskit-runtime-primitives-update) prior to submitting a circuit to IBM&reg; QPUs.
+    [Transpilation is necessary](/announcements/product-updates/2024-02-14-qiskit-runtime-primitives-update) prior to submitting a circuit to IBM&reg; QPUs.
 </Admonition>
 
 The steps in this topic describe how to transpile a given QASM circuit and obtain results using the Cloud Transpiler REST API, and include suggestions on how to submit the transpiled quantum circuit to IBM compute resources.  For an example that uses parameterized input, see [Sampler primitive with REST API and parameterized circuits.](/guides/primitives-rest-api#start-sampler-parms)
 
 ## Query the Qiskit Transpiler Service
 
-Query the [Qiskit Transpiler Service API](https://cloud-transpiler.quantum-computing.ibm.com/docs) and provide your QASM string as input. See more details in the [API reference documentation](/api/qiskit-transpiler-service-rest).
+Query the [Qiskit Transpiler Service REST API](https://cloud-transpiler.quantum-computing.ibm.com/docs) and provide your QASM string as input. See more details in the [API reference documentation](/api/qiskit-transpiler-service-rest).
```

Comment:
```suggestion
Query the Qiskit Transpiler Service REST API and provide your QASM string as input. See more details in the [API reference documentation](/api/qiskit-transpiler-service-rest) or view the [OpenAPI schema in Swagger UI](https://cloud-transpiler.quantum-computing.ibm.com/docs)
```

Reply:
Oops, I didn't realize I had an old link still. I meant to leave off the cloud-transpiler site because we don't want people leaving 1xp